egs/wsj/s5/utils/validate_lang.pl

```diff
@@ -814,22 +829,42 @@ sub check_summation {
       next;
     }
     $wlen = int(rand(100)) + 1;
+    $end_subword = 0;
```

Comment:
please add comments to clarify what you are doing here.  The previous script was not well documented, I know.

Reply:
Ok.